OpenRA.Mods.Common/Traits/Crates/Crate.cs

```diff
@@ -26,15 +29,18 @@ class CrateInfo : ITraitInfo, IPositionableInfo, Requires<RenderSpritesInfo>
 		[Desc("Allowed to land on.")]
 		public readonly HashSet<string> TerrainTypes = new HashSet<string>();
 
-		[Desc("Define actors that can collect crates by setting this into the Crushes field from the Mobile trait.")]
-		public readonly string CrushClass = "crate";
+		[Desc("Define actors that can collect crates by setting one of these into the Collects field from the CrateCollector trait.")]
+		public readonly BitSet<CollectorType> ValidCollectorTypes = new BitSet<CollectorType>("crate-collector");
+
+		[Desc("Set this to 'true' to allow crates to spawn on cells occupied by other actors. " +
+			"Note that this might make the crate unreachable if spawned on immobile, unremovable actors.")]
+		public readonly bool IgnoreActorsOnSpawning = false;
```

Comment:
What is the usecase for setting this `true`?

Reply:
Modders could want crates to be paradroppable on buildings. But I guess I should either drop this or add a valid collector check to avoid that 'unreachable' issue.